### PR TITLE
Dockerized platform

### PR DESCRIPTION
**Pull request by FallenChromium**
This PR depends on https://github.com/ostis-ai/sc-web/pull/10, https://github.com/ostis-ai/sc-machine/pull/140 and https://github.com/ostis-ai/sc-machine/pull/142
Changes:
- The whole Platform can be installed using Docker Compose, which is a big deal for users with unsupported OSes or hardware for native compilation.
- Launch scripts for native installation are updated to be in sync with latest changes in `sc-machine` and `sc-web`

### CODE REVIEW

**Review by SHtress — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <img src="https://github.com/ostis-ai/ostis-web-platform/actions/workflows/main.yml/badge.svg?branch=develop">
 
-### Documentation:
+# About:
```
Comment (NikitaZotov) [inline README.md:3]:
```suggestion
## About:
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - kb-binary:/kb.bin 
+    networks:
+      - sc-machine
+    ports:
+      - "8090:8090"
+    healthcheck:
+      test: "python3 /sc-machine/scripts/healthcheck.py"
+      interval: 10s
+      timeout: 10s
+      retries: 6
```
Comment (NikitaZotov) [inline docker-compose.yml:29]: Why 6?

File: README.md [hunk 3]
```diff
@@ -39,16 +56,14 @@ base to provide opportunity to use it in information processing and knowledge cr
...
-## Docker
-We're providing a Docker image to build the documentation. Head to [Installing with Docker](https://docs.docker.com/get-started/) 
-section of our docs to try it out!
+We're providing a Docker image to build the documentation.
 
 To do that, run docker image:
 ```sh
-cd ostis-web-platform/docs/scn/
-docker run -v </full/path/to/ostis-web-platform>:/workdir ostis/scntex-builder 'cd docs && pdflatex -interaction=nonstopmode main.tex'
+docker run -v <path/to/ostis-web-platform/>:/workdir ostis/scntex-builder 'cd docs && pdflatex -interaction=nonstopmode main.tex'
```
Comment (NikitaZotov) [inline README.md:63]: Is it right? May be it won't work, because this script must be started from ostis-web-platform/docs/scn/

**Review by SHtress — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (FallenChromium) [reply README.md:63]: 100% works, tested it myself. It doesn't matter as long as you're using the absolute path to `ostis-web-platform` to mount it

**Review by VasKho — APPROVED** [review]

**Review by NikitaZotov — APPROVED** [review]

**Review by KovalM — CHANGES_REQUESTED** [review]
Add section about build images

**Review by KovalM — APPROVED** [review]

**Pull request merged**